app/src/main/java/com/andrew/tasky/presentation/agenda_item_detail/AgendaItemDetailFragment.kt

```diff
@@ -38,16 +41,15 @@ class AgendaItemDetailFragment : Fragment(R.layout.fragment_agenda_item_detail)
         super.onViewCreated(view, savedInstanceState)
         binding = FragmentAgendaItemDetailBinding.bind(view)
         navController = Navigation.findNavController(view)
-        viewModel = ViewModelProvider(this).get(AgendaItemDetailViewModel::class.java)
+        viewModel = ViewModelProvider(this)[AgendaItemDetailViewModel::class.java]
         agendaItemType = AgendaItemType.valueOf(args.agendaItemType)
 
         subscribeToObservables()
         setupAgendaItemType()
         onClickListeners()
-
     }
 
-    fun <T> Fragment.collectLatestLifecycleFlow(flow: Flow<T>, onCollect: suspend (T) -> Unit) {
+    private fun <T> Fragment.collectLatestLifecycleFlow(flow: Flow<T>, onCollect: suspend (T) -> Unit) {
```

Comment:
If you follow my recommendation to move this into its own file like `FragmentExt.kt`, you can remove this one here